Description: skip failing nonet test
Author: Cédric Boutillier <boutil@debian.org>
Last-Update: 2020-07-31
Forwarded: no

--- a/test/test_document.rb
+++ b/test/test_document.rb
@@ -123,7 +123,7 @@
                  doc2.root.to_s(:indent => false))
   end
 
-  def test_nonet
+  def _test_nonet
     xml_string = '<ruby_array uga="booga" foo="bar"><fixnum>one</fixnum><fixnum>two</fixnum></ruby_array>'
     xml = LibXML::XML::Document.string(xml_string, options: LibXML::XML::Parser::Options::NONET)
     schema_document = LibXML::XML::Document.file('d:/src/libxml-ruby/test/model/atom.xml', options: LibXML::XML::Parser::Options::NONET)
